doc/enterprise/auth/auth.md

```diff
@@ -87,13 +87,13 @@ To activate access controls with `pachctl`, choose one of these options:
    ```bash
    $ pachctl auth activate --initial-admin=<prefix>:<user>
    ```
-  Note that you must prefix the username with the appropriate account
-  type, i.e. either `github:<user>`, or `robot:<user>`. If the latter
-  is used, Pachyderm will generate and return a Pachyderm auth token
-  that may be used to authenticate as the initial robot admin via
-  `pachctl auth use-auth-token` (see below). This can be useful in cases
-  where GitHub isn't an acceptable identity provider.
-  
+  **Note:** You must prefix the username with the appropriate account
+  type, such as `github:<user>` or `robot:<user>`. If you select the
```

Comment:
For this command the only two permitted prefixes are `github:` and `robot:`—I'm not sure if this is a technical writing best practice, but does it still make sense to say "such as" if there aren't any others?

Reply:
Nevermind, on re-reading the sentence, I now see that "such as" refers to the argument to `--initial-admin` rather than the set of possible prefixes